### PR TITLE
Text input playground

### PR DESCRIPTION
**Pull request by wenfeng-xing**
Screen Shot 2022-11-28 at 7 08 12 PM
Screen Shot 2022-11-28 at 7 08 22 PM
Screen Shot 2022-11-28 at 7 09 39 PM
Screen Shot 2022-11-28 at 7 09 49 PM

### CODE REVIEW

**Review by Honglin-Lu — APPROVED** [review]

File: docs/src/playground/SearchPage.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Grid, useTheme } from '@mui/material';
```
Comment (EdAdaptive) [inline docs/src/playground/SearchPage.tsx:1]: Is this the intended desktop view?  I didn't see a design in the Figma for one, and this differs greatly from the older versions.

File: docs/src/playground/SearchPage.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  { label: 'GBP - Great British Pound', type: 'currency' },
+  { label: 'USD - US Dollar', type: 'currency' },
+  { label: 'QOM - Shiba Predator', type: 'currency' },
+] as Option[];
+const groupBy = (opt: Option) => ({ stock: 'Stock', currency: 'FX' }[opt.type] as string);
+const SearchPage = () => {
+  const { palette } = useTheme();
+  const [input, setInput] = React.useState<string>('');
+
+  console.log('input', input);
```
Comment (EdAdaptive) [inline docs/src/playground/SearchPage.tsx:20]: Can remove the `console.log`

File: docs/src/playground/SearchPage.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      </Grid>
+      <Grid item xs={12} sx={{ height: '70vh' }}>
+        <TextSearch
+          options={options}
+          inputValue={input}
+          onInput={(event: any) => {
+            setInput(event.target.value);
+          }}
+          onChange={(event, value: Option | null) => {
+            console.log('event', event.target);
```
Comment (EdAdaptive) [inline docs/src/playground/SearchPage.tsx:44]: Can remove the `console.log`

**Review by EdAdaptive — CHANGES_REQUESTED** [review]

File: docs/src/playground/SearchPage.tsx (hunk 1 above)
Comment (wenfeng-xing) [reply docs/src/playground/SearchPage.tsx:1]: Thanks Ed, I'd forgotten that for desktop, we have a different layout :)

Comment (wenfeng-xing) [reply docs/src/playground/SearchPage.tsx]: Screen Shot 2022-11-30 at 7 29 25 AM
Screen Shot 2022-11-30 at 7 29 43 AM

**Review by EdAdaptive — APPROVED** [review]

**Pull request merged**